google-ads/build.gradle

```diff
@@ -369,7 +369,7 @@ publishing {
 
       groupId = 'com.google.api-ads'
       artifactId = 'google-ads-shadowjar'
-      version = '0.0.1-alpha'
+      version = '1.0.0'
```

Comment:
If this is ready to move out of the alpha phase, should the version match the overall library version? I'm concerned another versioning scheme could be confusing on top of the library and API versions.

Reply:
Good call, switched this over to use the module-wide versioning